flocker/testtools/_flaky.py

```diff
@@ -4,8 +4,19 @@
 Logic for handling flaky tests.
 """
 
+from functools import partial
+from pprint import pformat
 
-def flaky(jira_key):
+from pyrsistent import PClass, field, pmap, pset, pset_field
+import testtools
+from testtools.content import text_content
+from testtools.testcase import gather_details
+
+
+_FLAKY_ATTRIBUTE = '_flaky'
+
+
+def flaky(jira_keys, max_runs=5, min_passes=2):
```

Comment:
:grey_question: min_passes of 2 will always at least double the time to run for some of our large, considerably more flaky test suites, right? Personally I think defaults here of (max=3, min=1) would more accurately depict automating my current practices as an engineer.

Marking as optional because I'm guessing you've probably already had slack conversations and have some pretty good reason for believing 5/2 are good defaults for these values. I'm also fine with trying those out, but the wait time for AWS runners is already really long. Hopefully this decreases net wait time, but doubling the time for some of the tests seems as if it might threaten that.